pkg/tracker/index.go

```diff
@@ -165,13 +170,27 @@ func (i *IndexTracker) Exec(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	vals := make([]float64, 0)
 
-	vals, err := util.ParsePayload(payload, i.Args)
+	var decodedPayload interface{}
+	err = json.Unmarshal(payload, &decodedPayload)
+	if err != nil {
+		return err
+	}
+	match, err := jsonpath.Read(decodedPayload, i.JsonPath)
 	if err != nil {
 		return err
 	}
+	arr, ok := match.([]interface{})
+	if !ok {
+		return errors.Wrap(err, "JsonPath match need to be an array")
+	}
+	for _, a := range arr {
+		val, _ := strconv.ParseFloat(fmt.Sprintf("%v", a), 64)
+		vals = append(vals, val)
+	}
 
-	//fmt.Printf("got value of %f for %s\n", val, i.Identifier )
+	fmt.Fprintln(os.Stdout, fmt.Sprintf("got value of %f for %s\n", vals, i.Identifier))
```

Comment:
please use debug logging if you think that this is useful for debugging, otherwise please remove.

Reply:
Used for testing, I removed this now